project/Dependencies.scala

```diff
@@ -278,6 +279,15 @@ object Dependencies {
     "commons-codec" % "commons-codec" % commonsCodecV
   )
 
+  val languageFactoryCoreDependencies = List(
+  )
+
+  val wdlD2LanguageFactoryDependencies = List(
+  )
+
+  val cwlV1LanguageFactoryDependencies = List(
+  )
```

Comment:
These variables are unnecessary. Delete them from here and where they are currently used. The dependencies are an optional parameter. See `lazy val jesBackend` in build.sbt that doesn't have any extra dependencies.